app/main/errors.py

```diff
@@ -16,7 +15,7 @@ def page_not_found(e):
 
 
 @main.app_errorhandler(410)
-def page_not_found(e):
+def gone(e):
```

Comment:
Is this a flake8 fix? It's a more appropriate name but probably not in this PR.

I don't mind too much if you want to put this through here, but if you want to separate it to have better history that might be nice.